config/config.go

```diff
@@ -43,7 +43,7 @@ func DefaultConfig() *Config {
 		BaseConfig: BaseConfig{
 			LogLevel:              "info",
 			ValidatorMnemonic:     "",
-			DataEncryptionKeyFile: "config/data_encryption_key.sealed",
+			DataEncryptionKeyFile: "$HOME/.dataval/config/data_encryption_key.sealed",
```

Comment:
I recommend to use https://pkg.go.dev/os#UserHomeDir instead of `"$HOME"` for supporting various environments.

Reply:
I fixed in [this comment](https://github.com/medibloc/panacea-data-market-validator/pull/73#discussion_r851893321). Sorry for confusion.